packages/utils/src/Script.js

```diff
@@ -1,18 +1,47 @@
 /**
  * Tupaia
- * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 
-const { execSync } = require('child_process');
-const { existsSync, readdirSync } = require('fs');
-const { platform } = require('os');
+import { execSync } from 'child_process';
+import { existsSync, readdirSync } from 'fs';
+import { platform } from 'os';
 
-const { getArgs, getLoggerInstance } = require('@tupaia/utils');
+import { getLoggerInstance } from './getLoggerInstance';
+
+/**
+ * @typedef {Object} ScriptConfig
+ * @property {string} [command] Use '*' or '$0' to refer to the default command
+ *   This field is useful for
+ *   1. Defining subcommands, eg 'push <branch_name>'
+ *   2. Defining positional arguments in the default command, eg '* <path>'
+ * @property {Object} [options]
+ * @property {string} [usage]
+ * @property {string} [version]
+ *
+ * @see https://github.com/yargs/yargs/blob/master/docs/api.md
+ */
+
+/**
+ * @param {ScriptConfig} scriptConfig
+ */
+export const getArgs = scriptConfig => {
```

Comment:
This function (and the JsDoc above) are just moved here from the existing `script.js` file (now renamed to runScripts.js`)